pkg/controller/pd_control.go

```diff
@@ -14,802 +14,18 @@
 package controller
```

Comment:
how about focusing on design change in this PR and optimize later?

Reply:
okay, here is the PD API change: https://github.com/pingcap/tidb-operator/pull/618